{{cookiecutter.app_name}}/lib/web/assets/stylesheets/application.scss

```diff
@@ -2,8 +2,10 @@
 @import './variables';
 
 // Vendors
-{% if cookiecutter._bootstrap_addon == "yes" %}@import "./vendors/bootstrap";
-{% endif %}
+{% if cookiecutter._bootstrap_addon == "yes" %}@import "./vendors/bootstrap";{% endif %}{% if cookiecutter._tailwind_addon == "yes" %}@tailwind base;
+@tailwind components;
```

Comment:
Should we move these lines to `.vendors/tailwind` for consistency?

Reply:
Yes, makes sense!!